scripts/instantiate.bash

```diff
@@ -0,0 +1,153 @@
+#!/usr/bin/env bash
+
+NEUTRON_RPC="${NEUTRON_RPC:-tcp://0.0.0.0:26657}"
+NEUTRON_HOME="${NEUTRON_HOME:-$HOME/.neutrond}"
+NEUTRON_CHAIN_ID="${NEUTRON_CHAIN_ID:-test-1}"
+TARGET_CHAIN_ID="${TARGET_CHAIN_ID:-test-2}"
+GAS_PRICES="${GAS_PRICES:-0.005}"
+KEYRING_BACKEND="${KEYRING_BACKEND:-test}"
+DEPLOY_WALLET="${DEPLOY_WALLET:-demowallet1}"
+MIN_NTRN_REQUIRED="${MIN_NTRN_REQUIRED:-10}"
+ARTIFACTS_DIR="${ARTIFACTS_DIR:-../artifacts}"
+TARGET_SDK_VERSION="${TARGET_SDK_VERSION:-0.47.10}"
+TARGET_BASE_DENOM="${TARGET_BASE_DENOM:-uatom}"
+NEUTRON_SIDE_TRANSFER_CHANNEL_ID="${NEUTRON_SIDE_TRANSFER_CHANNEL_ID:-channel-788}"
+IBC_REGISTER_FEE="${IBC_REGISTER_FEE:-1000000}"
+INITIAL_VALIDATORS="${INITIAL_VALIDATORS:-[]}"
+
+
+source ./utils.bash
+
+echo "DEPLOY_WALLET: $DEPLOY_WALLET"
+echo "NEUTRON_RPC: $NEUTRON_RPC"
+
+
+main() {
+  set -euo pipefail
+  IFS=$'\n\t'
+
+  pre_deploy_check_code_ids
+  pre_deploy_check_balance
+  pre_deploy_check_ibc_connection 
+  deploy_factory
+  register_staker_ica
+  print_hermes_command $staker_ica_port $staker_ica_channel
+  wait_ica_address "staker" $staker_address
+
+  register_puppeteer_ica
+  print_hermes_command $puppeteer_ica_port $puppeteer_ica_channel
+  wait_ica_address "puppeteer" $puppeteer_address
+
+  update_msg='{
+    "update_config":{
+      "new_config":{
+        "puppeteer_ica":"'"$puppeteer_ica_address"'"
+      }
+    }
+  }'
+
+  msg='{
+    "wasm":{
+      "execute":{
+        "contract_addr":"'"$staker_address"'",
+        "msg":"'"$(echo -n "$update_msg" | jq -c '.' | base64)"'",
+        "funds": []
+      }
+    }
+  }'
+
+  factory_admin_execute $factory_address "$msg"
+  echo "[OK] Add Puppeteer ICA address to Staker contract config"
+
+  update_msg='{
+    "grant_delegate":{
+       "grantee":"'"$staker_ica_address"'"
+    }
+  }'
+
+  msg='{
+    "wasm":{
+      "execute":{
+        "contract_addr":"'"$puppeteer_address"'",
+        "msg":"'"$(echo -n "$update_msg" | jq -c '.' | base64)"'",
```

Comment:
```suggestion
        "msg":"'"$(echo -n "$update_msg" | jq -c '.' | base64 | tr -d "\n")"'",
```

By default, GNU base64 wraps output lines, which in the end leads to rendering a completely invalid JSON, making this script unusable on almost all Linux distributions.

I hate GNU and I wish it copied BSD behavior instead of doing its own stupid thing.

Reply:
Also same on line 53